### ISSUE TITLE
feat(linux): Debian package kmc

### ISSUE THREAD

**Issue by ermshiperete** [keymanapp/keyman#8527]
Include kmc compiler in Debian packaging.

**Comment by mcdurdin** [comment]
Note: .kmx are required for kmcmplib (see #8529), so we should not need to include kmc in the Debian packaging to get things to build -- which is a good thing.

If we do want kmc to be available as a Debian package (yes please!), it should be its own package, probably being built as part of a keyman-developer package (name to be confirmed). It has dependencies that Keyman itself really doesn't need -- e.g. node, emscripten.

**Comment by ermshiperete** [comment]
Yes, it will be a separate binary package, but build together with the other packages, i.e. in the same source package which is easier to maintain since it's all in the same repo. What we currently have is already split in several binary packages.

**Comment by ermshiperete** [comment]
This requires new enough nodejs and emscripten versions which we'll have to provide for most Ubuntu versions.

**Comment by ermshiperete** [comment]
Low priority since it can be installed with `npm`
